tests/components/wled/test_config_flow.py

```diff
@@ -59,6 +60,7 @@ async def test_show_zerconf_form(
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
 
 
+@patch("asyncio.sleep", new=AsyncMock())
```

Comment:
Why are we patching asyncio.sleep?

Reply:
Just wrote the answer below as an explanation on what updated 😄

Reply:
We try to avoid patching asyncio.sleep. That's pretty aggressive. It is better to patch the time slept. Easy to do if that time is a constant.

Since we seem to want to patch this thing in all tests, instead break it out ot a pytest fixture and set autouse to True.

Reply:
It is only patched on tests that test an error (as an error would trigger the backoff logic).

I've seen the time slept, however, time is not constant here (I really don't care about the time slept in this case either, as that is not in scope for Home Assistant).

Essentially the time slept logic/method would put in an `asyncio.sleep(0)` as a replacement. Not sure how that is an improvement over making is an `AsyncMock`? Both are awaited and return after being scheduled and done.

Reply:
That said, I fail to see a better solution in patching up the `backoff` decoration in this case.

Reply:
We don't want to patch asyncio since we use that in the core.

Reply:
Can we patch the backoff library function?

Reply:
Would it be acceptable to patch like so:

`@patch("backoff._async.asyncio.sleep", new=AsyncMock())` ?

Reply:
Patched out more locally, apparently, I was just thinking way too difficult 👯